src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -191,6 +193,7 @@ def _arm_to_cli_mapper(arm_dict):
             log_analytics_resource_id=arm_dict['logAnalyticsResourceId'] if 'logAnalyticsResourceId' in arm_dict else None),  # pylint: disable=line-too-long
         suffixes=CloudSuffixes(
             storage_endpoint=arm_dict['suffixes']['storage'],
+            storage_sync_endpoint=arm_dict['suffix']['storageSyncEndpointSuffix'] if 'storageSyncEndpointSuffix' in arm_dict['suffixes'] else 'afs.azure.net',  # pylint: disable=line-too-long
```

Comment:
```suggestion
            storage_sync_endpoint=arm_dict['suffix']['storageSyncEndpointSuffix'] if 'storageSyncEndpointSuffix' in arm_dict['suffixes'] else _get_storage_sync_endpoint(arm_dict['name']),  # pylint: disable=line-too-long
```
Can you add a method similar to `_get_microsoft_graph_resource_id` to return 'afs.azure.us' for US cloud and 'afs.azure.net' for other clouds?

Reply:
Sure, updated.